### PR TITLE
feat: Cleanup index page, fix model typos

### PR DESCRIPTION
**Pull request by jjmilburn**
**Index**

* Include reference to GH issues that ask for input on new resource
  types/models.
* Move verbose examples from intro to their own section

**Resource types**

* Add a few missing required fields from POST examples
* Modify conflicting example parameters
* Update energy consumption/generation suggested URI to not imply power flow direction related to the hosting device

(Updates are all semantic 'patch' updates, no new features or breaking changes)

### CODE REVIEW

File: resource_types/core/energy/102-generation/energygeneration.oas.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ produces:
   #- application/vnd.ocf+cbor
   - application/json
 paths:
-  "/ein":  # suggested URI
```
Comment (ricehornet) [inline resource_types/core/energy/102-generation/energygeneration.oas.yaml]: After thinking about this again, I realized I was still thinking about the convention wrong. Every device can just report the energy in and out. It's weird to report "this is the energy generated to me". So I'm reverting my request to change -- I think it's fine as `ein/eout`. Sorry for the confusion.

Comment (ricehornet) [reply resource_types/core/energy/102-generation/energygeneration.oas.yaml]: Also the `st` enum example still isn't a complete list?

**Comment by ricehornet** [comment]
@jjmilburn sorry for flip-flopping on the `ein/out` names -- I think I like those better actually. +1 once little suggestions are addressed.

File: resource_types/core/energy/102-generation/energygeneration.oas.yaml (hunk 1 above)
Comment (jjmilburn) [reply resource_types/core/energy/102-generation/energygeneration.oas.yaml]: Highlighting that changing back means that a solar panel would *not* host this resource (at least, the URI doesn't make as much sense then). So this is implying that the resource is hosted on a device that measures its *own* input, and energy generation devices don't report via this resource.

But thats just a convention with the URI, I guess, and we can always change the URI (on a solar panel, sure, just make the URI egen if you like)

Comment (ricehornet) [reply resource_types/core/energy/102-generation/energygeneration.oas.yaml]: Yes I understand that now! As long as we're consistent I'm fine -- and I'm sure we'll learn what's most useful when we get feedback from manufacturers.

**Pull request merged**